admin/tests/integration/components/menu-bar_test.js

```diff
@@ -7,13 +7,43 @@ module('Integration | Component | menu-bar', function (hooks) {
   setupRenderingTest(hooks);
 
   test('should contain link to "organizations" management page', async function (assert) {
```

Comment:
En lisant ces tests, je n'arrive pas à savoir si `link to "organizations" management page` est réservé seulement aux super admin ou si on a fait un seul test. 
Je trouverais ça plus lisible de mettre du contexte comme sur `module('Team tab', function () {`
C'est un peu hors scope de cette PR mais si tu as la connaissance en test ça serait cool je pense pour la suite pour savoir précisément les droits pour chaque élément du menu et que ça soit clair quand c'est juste les super admin qui voient un élément du menu. ^^

Reply:
Oui, on pourrait faire cette modification dans un ticket tech. C'est vrai que pour les parties où nous n'avons pas fait de modification ce n'est pas clair.